modules/efk-logging-deploy-subscription.adoc

```diff
@@ -15,38 +18,103 @@ You can use the {product-title} console to install cluster logging, which create
 
 . Make sure your environment meets the following requirements:
 +
-.. Ensure that you have deployed a router for the cluster.
-+
 .. Ensure that you have the necessary persistent storage for Elasticsearch. Note that each Elasticsearch node
 requires its own storage volume. 
 
 . Create a namespace for cluster logging:
+
+.. Create a YAML file with the following:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-logging
+  annotations:
+    openshift.io/node-selector: "" <1>
+  labels:
+    openshift.io/cluster-logging: "true"
+    openshift.io/cluster-monitoring: "true"
+----
+<1> Optionally specify an empty node selector in order for the logging pods to spread 
```

Comment:
The empty string `""` node selector means the logging pods will be spread evenly throughout the cluster.  If you want the logging pods to run on specific nodes, you can specify a node selector value here.